wavepool/templates/wavepool/newspost.html

```diff
@@ -19,7 +19,11 @@
 		<div class="col-2"></div>
 	</div>
 	<div class="newspost-detail">
-		<div class="row"><a href=""><button>edit</button></a></div>
+		{% if is_staff %}
+			<div class="row">
+				<a id="edit-link" href="{{ newspost.edit_url }}"><button>edit</button></a>
+			</div>
+		{% endif %}
```

Comment:
This conditional template shows `edit_url` property only when a logged in user has `is_staff` flag set. The `is_staff`flag is set into context in the `view.py`